src/biz/metrics_all_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2025 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !windows
+
+package biz
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/newrelic/infra-integrations-sdk/v3/persist"
+	"github.com/newrelic/nri-docker/src/raw"
+)
+
+// swap tests are not applicable for windows
```

Comment:
What about the specific metrics for windows? Do we need some sort of `metrics_windows_test` file?